dataset/app.py

```diff
@@ -13,27 +13,32 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+import typing
 
 from fastapi import FastAPI
 
 from dataset import __version__
 from dataset.config import get_settings
 from dataset.startup import api_registry
-from dataset.startup import create_app
 from dataset.startup import on_startup_event
 
-settings = get_settings()
-app: FastAPI = create_app(
-    title='Service Dataset',
-    description='Service Dataset',
-    debug=settings.DEBUG,
-    docs_url='/v1/api-doc',
-    version=__version__,
-)
 
-api_registry(app)
+def create_app(**app_kwargs: typing.Any) -> FastAPI:
```

Comment:
Suggest to remove since they are not used.
```suggestion
def create_app() -> FastAPI:
```